CreationalDesignPatterns/Builder/CoffeeShopDirector/ItalianBreakfastBuilder.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace CoffeeShopDirector
+{
+    public class ItalianBreakfastBuilder : IMenuBuilder

```

Comment:
We can return instance of the current object (this) in each of the methods. In this way we will be able to chain methods invocation. 
```
builder
.AddHotDrink()
.AddColdDrink()
.AddDessert();
```

Reply:
Can you provide an example?

Reply:
In this case an interface is used, not an abstract class for the builder

Reply:
I found a way, no problem 👍

Reply:
```
        public IMenuBuilder AddColdDrink()
        {	      
              this.menu.ColdDrink = Constants.DisplayJuice;
              return this;
         }
```